cmd/mdatagen/internal/templates/telemetry.go.tmpl

```diff
@@ -14,7 +14,6 @@ import (
     {{- end }}
 
 	"go.opentelemetry.io/otel/metric"
-    "go.opentelemetry.io/otel/metric/noop"
```

Comment:
Should this be its own PR?

Reply:
For you https://github.com/open-telemetry/opentelemetry-collector/pull/11197 :)

Reply:
appreciated!